app-config.yaml

```diff
@@ -99,7 +96,11 @@ scaffolder:
   # see https://backstage.io/docs/features/software-templates/configuration for software template options
 
 catalog:
-  locations: []
+  locations:
+    - type: url
+      target: https://dev.azure.com/santoshhkumarr/_git/REASSERT?path=/ado-templates/template.yaml
+      rules:
+        - allow: [Template]
```

Comment:
Is this repo public? otherwise this will not work without your sandbox's `AZURE_TOKEN`

Reply:
Looks private to me

Reply:
Public , share you with separately

Reply:
Thank you